release-linux-llvm/scripts/build-tiflash-release.sh

```
@@ -108,3 +108,49 @@ else
   # show version
   ${INSTALL_DIR}/tiflash version
 fi
+
+if [[ -n "${ENABLE_NEXT_GEN_COLUMNAR:-}" && "${ENABLE_NEXT_GEN_COLUMNAR}" != "false" && "${ENABLE_NEXT_GEN_COLUMNAR}" != "0" ]]; then
+  CMAKE_ENABLE_NEXT_GEN_COLUMNAR="ON"
+  echo "Building TiFlash with next-gen columnar features enabled"
+
+  # prepare build dir and install dir for columnar
+  if [ $CMAKE_BUILD_TYPE == "RELWITHDEBINFO" ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)